pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -39,8 +40,7 @@ func (h *Helper) InstallFlags(flags *pflag.FlagSet) {
 
 // GetDockerAuth returns a valid Docker AuthConfiguration entry, and whether it was read
 // from the local dockercfg file
-func (h *Helper) GetDockerAuth(registry, authType string) (docker.AuthConfiguration, bool) {
-	var authCfg docker.AuthConfiguration
+func (h *Helper) GetDockerAuth(imageName, authType string) (docker.AuthConfiguration, bool) {
```

Comment:
`registry` to `imageName`?  I suspect this change in meaning need to rippled back to callers?